fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/FedoraResourceImplTest.java

```diff
@@ -395,6 +398,14 @@ public void testGetProperty() throws RepositoryException {
     }
 
     @Test
+    public void testSetURIProperty() throws URISyntaxException, RepositoryException {
+        final String prop = "premis:hasEventRelatedObject";
+        final String uri = "http://localhost:8080/rest/1";
```

Comment:
The URIs are references to things in the repo -- identifiers of resources related to audit events.  The intention with this approach is to build those URIs in the audit processor, and store them in the repository as URIs, so they are not deleted if the resource is deleted.